databrowse/plugins/db_PDF_viewer/handlers.py

```diff
@@ -32,7 +32,14 @@
 ## performed at Iowa State University.                                       ##
 ##                                                                           ##
 ## DISTRIBUTION A.  Approved for public release:  distribution unlimited;    ##
-## 19 Aug 2016; 88ABW-2016-4051.											 ##
+## 19 Aug 2016; 88ABW-2016-4051.                                             ##
```

Comment:
There is no need to add this language to files that you did not otherwise modify.  It is potentially useful to keep track of what work has been done on what projects.